src/json_schema_factory.coffee

```diff
@@ -35,11 +36,33 @@ buildDefinitionFromConfig = (config, propertyIsRequired) ->
     when config.type is 'boolean'
       -> faker.random.array_element [true, false]
 
-    when config.format is 'date-time'
-      -> faker.date.between(new Date('2013-01-01'), new Date('2014-01-01'))
-
     when config.type is 'string'
-      -> faker.lorem.words().join ' '
+      switch config.format
+        when undefined
+          -> faker.lorem.words().join ' '
+
+        # see https://github.com/goodeggs/goodeggs-json-schema-validator for supported formats
+        when 'objectid'
```

Comment:
FYI goodeggs-json-schema-validator is definitely still a private module! If you make it public, then carry on :+1: